single-node-refactor/src/Solvers/SGH_solver/include/contact.h

```diff
@@ -0,0 +1,148 @@
+#pragma once
+
+#include "matar.h"
+#include "mesh.h"
+#include "_debug_tools.h"  // Remove this file entirely once finished
+
+using namespace mtr;
+
+struct contact_patch_t
+{
+    size_t gid;  // global patch id
+    CArrayKokkos<size_t> nodes_gid;  // global node ids
+
+    /*
+     * If the position of a point is denoted by "p" and "p" is a vector p = (px, py, pz), then the following arrays
```

Comment:
I like this type of documentation.  Thanks!

Reply:
No problem!